knowledge-base/common-icon-names-list.md

```diff
@@ -12,7 +12,7 @@ res_type: kb
 
 ## Description
 
-Since the `IconName` class is marked as obsolete and will soon be removed, below you will find a valid list of icon names to be hardcoded.
+If you have used the obsolete `IconName` class, here is a list of icon names in its stead.
```

Comment:
maybe use "icon names" as a link to the current article, or even expand with a sentence that one shoud use strings now and that they are available in this and that article